clients/java/zms/src/main/java/com/yahoo/athenz/zms/ZMSClient.java

```diff
@@ -190,19 +193,11 @@ public void close() {
     }
 
     /**
-     * Set new ZMS Client configuration property. This method calls
-     * internal javax.ws.rs.client.Client client's property method.
-     * If already set, the existing value of the property will be updated.
-     * Setting a null value into a property effectively removes the property
-     * from the property bag.
-     *
-     * @param name  property name.
-     * @param value property value. null value removes the property with the given name.
+     * Set the DNSResolver to be used by the ZMS Client
+     * @param resolver user supplied dns resolver
      */
-    public void setProperty(String name, Object value) {
```

Comment:
This public method was removed from ZMSClient which was probably used by existing clients of the library. Once they will update to the new version their code will not compile. 
If this function can be safely ignored we can consider marking it as Deprecated and have it do nothing. Otherwise we should consider updating the README file with the API changes in the new version.